docs/guides/implementing-connectors.md

```diff
@@ -0,0 +1,88 @@
+
+### Create a cluster
+
+import CreateCluster from './assets/react-components/create-cluster.md'
+
+## Getting started
+
+Once logged in to your Camunda Platform 8 account, take the following steps:
+
+1. Click on the **Modeler** tab at the top of the page, and click **New project**.
+2. Name your project. In this example, we'll name ours `Expense process`.
+3. Select **New > BPMN Diagram > + Create blank**.
+4. Give your model a descriptive name, and then give your model a descriptive id within the **General** tab inside the properties panel on the right side of the screen. In this case, we've named our model `Submit expense` with an id of `submitting-expense`.
+
+## Build a BPMN diagram
+
+Use Web Modeler to design a BPMN flow with the appropriate tasks. To get started, create a task by dragging the rectangular task icon from the palette, or click the existing start event and the displayed task icon.
+
+In this example, we've designed the following BPMN diagram:
+
+![bpmn example diagram](./img/bpmn-expense-sample.png)
+
+[//]: # 'Do you want me to add steps for building out the sample diagram here? Or, would it be better to say something like "To learn more about building your own BPMN diagram from scratch, visit our guide on automating a process using BPMN."'
+[//]: # 'Do we want to change the diagram so the start event is "Receipt ready" and the first task is "Receipt uploaded?"'
+
+## Add a connector
+
+Here, a receipt is initially uploaded for review. The first task we need to complete is notifying the manager of the uploaded receipt. If we want to leverage our email service to notify the manager, we can utilize a productivity applications connector to replace this task.
+
+:::note
+Camunda offers a variety of available connectors and even has the opportunity to construct custom connectors. For example, utilize cloud connectors to communicate with cloud-native applications and conform to REST, GraphQL, or SOAP protocols. Or, employ service connectors to integrate with technology enablers like RPA, AI or IOT services. Learn more about our [available connectors](../components/modeler/web-modeler/connectors/available-connectors/index.md) to find out which may best suit your business needs.
+:::
+
+To add our productivity applications connector, take the following steps:
+
+1. Click the start event. A panel to the right of the start event will appear.
+2. Click the **Append connector** item in the panel.
+3. To send an email via SendGrid, for example, select the **SendGrid Email Connector** option. Name this newly-created task `Notify manger of receipt`. This now replaces our original task.
+   ![adding a connector](./img/adding-connector.png)
+4. You need to fill out the required information in the properties panel of this task on the right side of the screen. Here, we'll add an example SendGrid API key, a sender and receiver name and email address, and the email message content.
```

Comment:
You may need to make this clearer that the API key is available through SendGrid, not Camunda.

Reply:
Updated in latest commit.